src/libraries/System.Private.CoreLib/src/System/Numerics/BitOperations.cs

```diff
@@ -218,6 +238,23 @@ public static int LeadingZeroCount(ulong value)
             return LeadingZeroCount(hi);
         }
 
+        /// <summary>
+        /// Count the number of leading zero bits in a mask.
+        /// Similar in behavior to the x86 instruction LZCNT.
+        /// </summary>
+        /// <param name="value">The value.</param>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [CLSCompliant(false)]
+        public static int LeadingZeroCount(nuint value)
+        {
+            if (Environment.Is64BitProcess)
```

Comment:
Could you do the same `#if TARGET_64BIT` to all of these `if (Enviroonment.Is64BitProcess)` paths?

Reply:
Done